repo/packages/C/confluent-rest-proxy/3/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "3.0",
+  "name": "confluent-rest-proxy",
+  "version": "0.9.6-3.1.1",
+  "scm": "https://github.com/confluentinc/kafka-rest",
+  "description": "Confluent REST Proxy service",
```

Comment:
Adding link to existing Confluent docs for now; link to dcos examples when complete.